integration_test/build-adapters.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# imageProject commitShortSha
+
+base_name="gcr.io/${1}/healthcare-api-dicom-dicomweb-adapter-"
+
+build_adapter () {
+  gradle dockerBuildImage
+  adapter_name=${base_name}${1}
+  version=$(gradle printVersion  | grep -Eo "[0-9]{1,3}\.[0-9]{1,3}" | head -1)
+  postfix=$(gcloud container images list-tags "${adapter_name}" --sort-by=~timestamp | grep " $version" |  grep -Eo "([0-9]{1,3}\.){2}[0-9]{1,3}" | head -1 | grep -Eo "[0-9]{1,3}" | tail -1)
+  postfix=$(($postfix+1))
+  gradle dockerBuildImage -Pdocker_tag=${adapter_name}:$version.$postfix
+  gradle dockerBuildImage -Pdocker_tag=${adapter_name}:$2
+}
+apt-get -qq update
+apt-get install -y gnupg2
+echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
```

Comment:
Can you add a comment explaining what these lines do and why they're needed?

Reply:
done, after recent update we've moved to java 11 so we need gcloud and gradle 5.6 with jdk11 in same image. standard ubuntu repo don't have java 11, gradle 5 and gcloud.
gcloud is smaller and packages hosted by google so i've decided to use gradle's image and install gcloud by this manual https://cloud.google.com/sdk/docs/quickstart-debian-ubuntu